src/meta/processors/indexMan/RebuildTagIndexProcessor.cpp

```diff
@@ -0,0 +1,92 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "meta/ActiveHostsMan.h"
+#include "meta/processors/admin/AdminClient.h"
+#include "meta/processors/indexMan/RebuildTagIndexProcessor.h"
+
+namespace nebula {
+namespace meta {
+
+void RebuildTagIndexProcessor::process(const cpp2::RebuildIndexReq& req) {
+    auto space = req.get_space_id();
+    CHECK_SPACE_ID_AND_RETURN(space);
```

Comment:
CHECK_SPACE_ID_AND_RETURN should be before get_space_id.

Reply:
CHECK_SPACE_ID_AND_RETURN  use space id as parameter